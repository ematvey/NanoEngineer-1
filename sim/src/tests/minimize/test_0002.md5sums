stdout 9961762F03FC62301F295D9630F78043
stderr D41D8CD98F00B204E9800998ECF8427E
