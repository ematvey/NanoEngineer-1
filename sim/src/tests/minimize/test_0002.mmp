mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (0.931707, 0.283577, 0.061527, -0.218450) (7.290000) (0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (test)
mol (Chunk-C1) tub
atom 1 (6) (-799, 5348, -1559) def
atom 2 (0) (-189, 5715, -1264) def
bond1 1
atom 3 (0) (-1167, 5814, -2048) def
bond1 1
atom 4 (6) (-5012, 84, -135) def
bond1 1
atom 5 (0) (-5115, -538, -576) def
bond1 4
atom 6 (0) (-5654, 225, 264) def
bond1 4
atom 7 (6) (-72, 739, 1778) def
bond1 4
atom 8 (0) (42, 806, 2536) def
bond1 7
atom 9 (0) (256, 1332, 1414) def
bond1 7
atom 10 (6) (816, 1628, -889) def
bond1 1
atom 11 (0) (121, 1684, -562) def
bond1 10
atom 12 (0) (841, 998, -1331) def
bond1 10
atom 13 (6) (7742, 2455, 696) def
bond1 10
atom 14 (0) (7988, 3042, 1130) def
bond1 13
atom 15 (0) (8326, 2189, 270) def
bond1 13
atom 16 (6) (3146, -1161, 2167) def
bond1 13 7
atom 17 (0) (3572, -1589, 2645) def
bond1 16
atom 18 (0) (2515, -1554, 1966) def
bond1 16
egroup (test)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part test
