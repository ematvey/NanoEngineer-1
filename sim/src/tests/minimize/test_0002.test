TYPE fail
PROGRAM simulator -f500 -m -x test_0002.mmp

