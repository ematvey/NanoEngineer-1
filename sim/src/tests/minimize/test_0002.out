======= test_0002.test =======
TYPE fail
PROGRAM simulator -f500 -m -x test_0002.mmp

